x-pack/plugins/security_solution/public/flyout/right/components/investigation_section.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { VFC } from 'react';
+import React from 'react';
+import { ExpandableSection } from './expandable_section';
+import { HighlightedFields } from './highlighted_fields';
+import { INVESTIGATION_SECTION_TEST_ID } from './test_ids';
+import { INVESTIGATION_TITLE } from './translations';
+
+export interface DescriptionSectionProps {
+  /**
+   * Boolean to allow the component to be expanded or collapsed on first render
+   */
+  expanded?: boolean;
```

Comment:
like I answered [here](https://github.com/elastic/kibana/pull/154196/files#r1156450316), I like have the props colocated with the component, and I prefer writing identical ones instead of sharing them... Would you be opposed to that?